### PR TITLE
Doc: update README.md

### PR DESCRIPTION
**Pull request by MaikoTan**
since jobs module was changed a lot

[Click Here](https://github.com/MaikoTan/cactbot/blob/readme-update/README.md#jobs-module) to see the page.

### CODE REVIEW

**Comment by Echoring** [comment]
↑Done.

File: README.md [hunk 1]
```diff
@@ -341,14 +341,36 @@ or raiding, and a visual pull countdown.
...
 
-* Red Mage: Shows white/black mana, tracks procs for Verstone, Verfire and Impact, and shows the state of the melee combo in progress.
-* Warrior: Shows the beast amount, and tracks the remaining Storm's Eye buff time in gcds.
-* Monk: Shows chakra count, remaining greased lightning time, and tracks monk buffs and debuffs.
+<details>
+<summary>Supported Jobs (Click to expand)</summary>
+
+|Job|Feature|
+|:-:|:-:|
+|<img src="https://xivapi.com/cj/1/paladin.png" width="30px"/><br> Paladin|Shows current Oath amount, and atonement stacks. Also tracks Goring Blade DoT. |
```
Comment (quisquous) [inline README.md:349]: I think I would prefer not to hotlink from xivapi.  Can you use abbreviations (e.g. PLD) or copy icons locally (docs/images/? somewhere else?)

**Review by quisquous — COMMENTED** [review]
Thanks for the doc update!

File: README.md (hunk 1 above)
Comment (MaikoTan) [reply README.md:349]: OK, I think put into local is good.

**Comment by MaikoTan** [comment]
cc @Una-p @Jaehyuk-Lee 
Just let you know that `README.md` was updated, so Korean version should be updated too.

**Pull request merged**